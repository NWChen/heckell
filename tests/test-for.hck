let sum: int;
sum = 0;
for x in [0, 2, 4, 5] do
    sum = sum + x;
    print("\(sum)");
end