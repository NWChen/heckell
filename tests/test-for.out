0
2
6
11
